client/src/pages/JoinPage.tsx

```diff
@@ -56,6 +57,17 @@ const JoinPage = ({
     setGain(_gain);
   };
 
+  function Visualizer() {
```

Comment:
I think this should just its own component maybe with a more specific name like AudioVisualizerBar or something

Reply:
100% agree. Did you see my branch where I moved the buttons bar in the main canvas area to it's own component as well? I want to do a couple refactors like that. Plus this can be made a little more modular by expressing it's width as a percentage in case we want a smaller one somewhere

Reply:
I made an issue for this refactor that I can do pretty quickly, but in the spirit of this PR I think it should be separate

Reply:
Oh I just saw it perfect nice!
Yeah you're right, it should be handled in a separate ticket.